### ISSUE TITLE
vue3 based template

### ISSUE THREAD

**Issue by softboy99** [DevExpress/devextreme-vue-template#12]
vue3 based template

**Comment by LexDX** [comment]
Hi @softboy99,

At the moment, we are working on the integration of our components in Vue 3 in the context of this PR: https://github.com/DevExpress/devextreme-vue/pull/383. Once we finish the integration, we have plans to update our Project Template to Vue 3. We will announce these updates in our Blog Posts and What's New articles, so I recommend you stay tuned to them. Also, please check our Project Template repository updates.

**Comment by softboy99** [comment]
@LexDX , "npx -p devextreme-cli devextreme new vue-app app-name"  still use v20.1.7 version and it still based on vue2.x

**Comment by LexDX** [comment]
@softboy99 ,

We introduced support for Vue 3 in v.20.2.3. However, in this version, we also had an issue with our project templates. So, we did not change the default version for our template in our CLI tools. In v.20.2.4 we fixed this issue and will update our DevExtreme CLI, so it will generate a template for v.20.2.4 in a couple of days if it all goes as expected.

At the same time, I'd like to note that our template still uses Vue 2 as we did not upgrade it to Vue 3 yet. But, we have plans to introduce such a template in the near future. Please stay tuned to our Blog posts for updates.

**Comment by softboy99** [comment]
@LexDX  your v20.2.4 application template still use vue2.x, after mannually changed to vue3.x, it cannot be runned any more.

**Comment by LexDX** [comment]
@softboy99 ,

We are still working on the Vue 3 template. 
As for the manual conversion, we do not have a migration guide for Vue 3 as our controls should work fine without any specific changes in their code. However, it is necessary to migrate the application itself. Thus, I recommend you use the standard Vue 2 to Vue 3 migration guides. For example, I found the following articles:
[The Vue 3 upgrade guide](https://dev.to/blacksonic/the-vue-3-upgrade-guide-4dc4)
[How to migrate your library from Vue2 to Vue3](https://dev.to/alvarosaburido/how-to-migrate-your-library-from-vue2-to-vue3-1h81)

**Comment by softboy99** [comment]
Hi,
@LexDX ,when the vue3 template will be available? what's the plan? v20.2.5 or v21.1?

**Comment by LexDX** [comment]
Hi @softboy99,

I'm happy to inform you that we released our Vue 3 template a couple of weeks ago. Now, you can use our CLI to install and try it on your side:
```js
npx -p devextreme-cli devextreme new vue-app app-name --version=3
```
[Create a DevExtreme Application][0]

Should you have further questions, feel free to contact us. We are here to help.

[0]: https://js.devexpress.com/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/
